zhaquirks/tuya/ts011f_plug.py

```diff
@@ -1557,3 +1557,64 @@ class Plug_TZ3000_2AC_var02(EnchantedDevice):
             },
         },
     }
+
+
+class TZ3000_Socket(CustomDevice):
+    """Tuya Socket with child lock and power-restore state support."""
+
+    quirk_id = TUYA_PLUG_ONOFF
+
+    signature = {
+        MODELS_INFO: [("_TZ3000_o1jzcxou", "TS011F")],
```

Comment:
I think it is worth using the `MODEL: "TS011F",` format here for this quirk as well.